notebook/converted/benchmark_results.py

```diff
@@ -1115,13 +1115,61 @@ def return_unlabeled():
 tissue_metrics
 
 
-# In[8]:
+# In[4]:
 
 
 plot = ggplot(tissue_metrics, aes(x='train_count', y='balanced_accuracy', color='supervised')) 
 plot += geom_smooth()
 plot += geom_point(alpha=.2)
 plot += geom_hline(yintercept=.5, linetype='dashed')
-plot += ggtitle('Brain vs Breast Tissue Prediction')
+plot += ggtitle('Blood vs Breast Tissue Prediction')
 plot
 
+
+# ### All Tissue Predictions
+
+# In[5]:
+
+
+in_files = glob.glob('../../results/all-tissue.*.tsv')
+print(in_files[:5])
+
+
+# In[6]:
+
+
+tissue_metrics = pd.DataFrame()
+for path in in_files:
+    new_df = pd.read_csv(path, sep='\t')
+    model_info = path.strip('.tsv').split('all-tissue.')[-1]
+    model_info = model_info.split('_')
+        
+    supervised_model = '_'.join(model_info[:2])
+             
+    new_df['supervised'] = supervised_model
+    
+    new_df['seed'] = model_info[-1]
+        
+    tissue_metrics = pd.concat([tissue_metrics, new_df])
+    
+tissue_metrics['train_count'] = tissue_metrics['train sample count']
+
+tissue_metrics['supervised'] = tissue_metrics['supervised'].str.replace('pytorch_supervised', 'three_layer_net')
+tissue_metrics
+
+
+# In[9]:
+
+
+plot = ggplot(tissue_metrics, aes(x='train_count', y='balanced_accuracy', color='supervised')) 
```

Comment:
I think you've done similar things in past experiments and I'm probably forgetting, but what is the classification task here? Are you doing something like one-vs-rest classification for each tissue type individually, or are you doing multiclass classification (without binarizing) and calculating accuracy over all classes?

My guess is the latter (since accuracy scores are lower overall than for the binary tissue classification task), but I just want to make sure my understanding is correct.

Reply:
This is multiclass classification across all classes, and the metric is balanced accuracy (the mean of the recall of all classes). That is to say, you are correct :)